### ISSUE TITLE
Numbas exam fails to open with TypeError: "tok" of null

### ISSUE THREAD

**Issue by oscarsiles** [numbas/Numbas#684]
I have had a student receive the following error on one of their attempts, which does not allow the attempt to be run by either themselves or a teacher. Their scores and JSON data can be seen just fine in the LTI tool.

`TypeError: Cannot read property 'tok' of null
at Object.jme.findvars (xxx/scripts.js:21064:13)
at c. (xxx/scripts.js:28766:36)
at Function.Uc (xxx/scripts.js:22771:20)
at Function.Vc (xxx/scripts.js:22770:299)
at Function.U (xxx/scripts.js:22770:126)
at xxx/scripts.js:22769:462
at xxx/scripts.js:22735:297`

Not quite sure what might have caused this, to be honest, as the student was saving and returning to the exam a handful of times without issues. It does appear the student was able to submit a blank answer for one of the questions during the session that caused the issue to appear.

Am happy to provide additional information if needed

**Comment by christianp** [comment]
Can you give a link to the exam in the Numbas editor please?

**Comment by christianp** [comment]
Didn't meant to close this! Looking at the error message, I made a guess that somehow `Numbas.jme.findvars` was being passed a `null` value, which I suppose could happen if the student had submitted a blank answer. I've added a line to catch this safely.

Now I'll try to work out how the student submitted nothing.

**Comment by christianp** [comment]
OK, I worked out how to reproduce the problem, and I believe I've fixed it. In order for this student to continue their attempt, you'd have to update the exam package. I've just added an option to the LTI tool to make existing attempts use the new version when you replace the exam package; your admin will need to update your installation of the LTI tool to [version 2.6](https://github.com/numbas/numbas-lti-provider/releases/tag/v2.6).
